### PR TITLE
completes endpoints and orders data from db returns for rooms and use…

### PR DESCRIPTION
**Pull request by jamesjlundin**
…rs tables

## PR TITLE
**Description:** 
  
room model and users model returns data sorted by the names or rooms and users, also fixes bug in endpoint for updating a user role
  
to complete functionality for updating user roles and deleting rooms
  
allows frontend to hit endpoints for updating user roles and deleting rooms
  
no

### CODE REVIEW

**Review by jakegrella — APPROVED** [review]
looks good, good idea on the ordering

File: models/admin.js [hunk 1]
```diff
@@ -32,6 +31,7 @@ const roleDelete = (role_id) => {
 };
 
 module.exports = {
+  fetchUser,
```
Comment (salza323) [inline models/admin.js:34]: Looks like this was prob the issue when not being able to pull users in admin settings.

**Review by salza323 — APPROVED** [review]
Nice! See note in admin model.

**Review by JustinMrks — APPROVED** [review]
Agreed, great idea to find users much easier

**Pull request merged**